### PR TITLE
[py-aiohttp] added new version

### PR DESCRIPTION
**Pull request by mdorier**
New versions for the py-aiohttp package.

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-aiohttp/package.py [hunk 1]
```diff
@@ -13,7 +13,13 @@ class PyAiohttp(PythonPackage):
...
+    version('3.8.1', sha256='45210606552ed4f8992e17ffa3273e9ad2815c8c3ca00832a57b3a62735b1fc7')
+    version('3.8.0', sha256='0587288bf063cc0433542ab17674af11cb3639f4cb5ddf89420501ca16565e75')
     version('3.7.4', sha256='d90ace66b55747e49531b13bf0a9c7ae9d1e14f315a56ea6b54c0dc1b6facd6e')
+    version('3.7.3', sha256='2caac02447ff5d5b2db30a828fe676e90bb01c384e7751392784b68b7c7fcc18')
+    version('3.7.2', sha256='22ed7395d53c03f0f4872e5cdd051330bbce33f3d81b0af9aefd48a7077db040')
+    version('3.7.1', sha256='6075f4fb46cfb73f9282709272ec4755967668c633b6276433f4bad5b6db1b2b')
+    version('3.7.0', sha256='4549b9f1857dc2f8ba4028d8ba697eb7d7e5db7803fc7eee232f07027551ea26')
     version('3.6.2', sha256='4ad2d8393843ea0c7b50a93bf04fb1cf20b93c0a5e958de128efcd2e5f8adf80')
 
     depends_on('py-setuptools', type='build')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-aiohttp/package.py]:
```suggestion
    depends_on('py-setuptools@46.4:', type='build')
```

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-aiohttp/package.py]: Other deps need to be updated too, see `setup.cfg`

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Review by adamjstewart — APPROVED** [review]

**Comment by mdorier** [comment]
I think there were still dependencies to be added.

**Comment by adamjstewart** [comment]
Ah yes, there are definitely missing deps, can you submit another PR?

**Comment by mdorier** [comment]
I was looking into it this afternoon but saw that some deps aren't yet in spack (or at least not a recent-enough version), and got discouraged. I need to find the time to look again into it.

**Comment by adamjstewart** [comment]
We could also revert this PR if you don't have time to add the deps and don't want a broken package in develop.

**Comment by mdorier** [comment]
I think that would be best, it's going to take me a few days before I can circle back to it. Thanks!

**Comment by mdorier** [comment]
Updating the py-aiohttp package with correct dependencies for the new version will require the following PRs to be merged:
- https://github.com/spack/spack/pull/28194
- https://github.com/spack/spack/pull/28195

**Comment by adamjstewart** [comment]
Just so you know, we used to have a policy where we tried to keep each PR to only a single package, but we've recently relaxed this so that packages that depend on changes in another package can be bundled in a single PR. You're welcome to continue submitting separate PRs for each change, but you don't have to anymore.

**Comment by mdorier** [comment]
That's good to know!

**Pull request merged**